tools/benchmark/src/main/java/com/datadog/benchmark/internal/BenchmarkSpanToSpanEventMapper.kt

```diff
@@ -20,7 +19,7 @@ internal class BenchmarkSpanToSpanEventMapper {
         val durationNanos = spanData.endEpochNanos - spanData.startEpochNanos
 
         return SpanEvent(
-            traceId = spanData.traceId,
+            traceId = spanData.traceId.substring(0, SPAN_EVENT_TRACE_ID_LENGTH),
```

Comment:
very minor: or simply `take` should work

```suggestion
            traceId = spanData.traceId.take(SPAN_EVENT_TRACE_ID_LENGTH),
```

Reply:
updated